nms/app/packages/magmalte/app/state/lte/SubscriberState.js

```diff
@@ -238,3 +247,89 @@ export function getSubscriberGatewayMap(subscribers: {[string]: subscriber}) {
   });
```

Comment:
Hey @HannaFar , Given that we can use entire subscriber_state, can you use the subscriber state to get the directory and location history for the subscriber ? That also contains similar fields. Following I have attached subscriber and subscriber state json for reference
Subscriber response
https://www.internalfb.com/phabricator/paste/view/P393148630

Subscriber state response
https://www.internalfb.com/phabricator/paste/view/P393151756